compiler/rustc_symbol_mangling/src/v0.rs

```diff
@@ -93,6 +93,26 @@ pub(super) fn mangle_typeid_for_fnabi<'tcx>(
     format!("typeid{}", arg_count)
 }
 
+pub(super) fn mangle_typeid_for_trait_ref<'tcx>(
+    tcx: TyCtxt<'tcx>,
+    trait_ref: ty::PolyExistentialTraitRef<'tcx>,
+) -> String {
+    // FIXME(flip1995): Seem comment in `mangle_typeid_for_fnabi`.
+    let prefix = "_ZTS";
```

Comment:
I don't think this needs a specific prefix. Rust vtables aren't compatible with C++ anyway.